src/main/java/it/pagopa/selfcare/external_api/mapper/OnboardingMapper.java

```diff
@@ -53,7 +52,13 @@ public interface OnboardingMapper {
       target = "contractImported.activatedAt",
       source = "contractImported.activatedAt",
       qualifiedByName = "convertDate")
-  OnboardingImportRequest toOnboardingImportRequest(OnboardingData onboardingData);
+  OnboardingImportRequest mapToOnboardingImportRequest(OnboardingData onboardingData);
+
+  default OnboardingImportRequest toOnboardingImportRequest(OnboardingData onboardingData) {
+    OnboardingImportRequest onboardingImportRequest = mapToOnboardingImportRequest(onboardingData);
+    onboardingImportRequest.getInstitution().setOrigin(Origin.IPA);
```

Comment:
it should be better if this can be overwritten

Reply:
done